packages/api2/src/database/migrations/31_settings_add_label_description.ts

```diff
@@ -0,0 +1,124 @@
+import { SettingModel } from '../schemas/settings/00_settings.schema';
+
+const settings = [
```

Comment:
The migration should only affect `PRAISE_QUANTIFIERS_PER_PRAISE_RECEIVER`. Now you include a whole list of settings that are not needed. Remove everything but `PRAISE_QUANTIFIERS_PER_PRAISE_RECEIVER`.

Reply:
Updated the PR, please check now